docs/concepts.rst

```diff
@@ -309,87 +309,85 @@ The :class:`~pystac.validation.JsonSchemaSTACValidator` takes a :class:`~pystac.
 Extensions
 ==========
 
-Accessing Extension functionality
----------------------------------
-
-All STAC objects are accessed through ``Catalog``, ``Collection`` and ``Item``, and all extension functionality
-is accessed through the ``ext`` property on those objects. For instance, to access the band information
-from the ``eo`` extension for an item that implements the extension, you use:
+From the documentation on `STAC Spec Extensions 
+<https://github.com/radiantearth/stac-spec/tree/master/extensions>`__:
 
-.. code-block:: python
+   Extensions to the core STAC specification provide additional JSON fields that can be
+   used to better describe the data. Most tend to be about describing a particular 
+   domain or type of data, but some imply functionality.
 
-   # All of the below are equivalent:
-   item.ext['eo'].bands
-   item.ext[pystac.Extensions.EO].bands
-   item.ext.eo.bands
+This library makes an effort to support all extensions that are part of the
+`stac-extensions GitHub org
+<https://stac-extensions.github.io/#extensions-in-stac-extensions-organization>`__, and
+we are committed to supporting all STAC Extensions at the "Candidate" maturity level or
+above (see the `Extension Maturity
+<https://stac-extensions.github.io/#extension-maturity>`__ documentation for details).
 
-Notice the ``eo`` property on ``ext`` - this utilizes the `__getattr__ <https://docs.python.org/3/reference/datamodel.html#object.__getattr__>`_ method to delegate the property name to the ``__getitem__`` method, so we can access any registered extension as if it were a property on ``ext``.
+Accessing Extension Functionality
```

Comment:
Should include a section on adding extensions to STACObjects e.g. `EOExtensions.add_to(item)`

Reply:
Good point. Added this in 6d297b346e42086db00d1a7e30564dc3611fbca7